LearningHub.Nhs.WebUI.AutomatedUiTests/AccessibilityTests/AccessibilityTestsBase.cs

```diff
@@ -0,0 +1,67 @@
+// <copyright file="AccessibilityTestsBase.cs" company="HEE.nhs.uk">
+// Copyright (c) HEE.nhs.uk.
+// </copyright>
+
+namespace LearningHub.Nhs.WebUI.AutomatedUiTests.AccessibilityTests
+{
+    using FluentAssertions;
+    using LearningHub.Nhs.WebUI.AutomatedUiTests.TestFixtures;
+    using LearningHub.Nhs.WebUI.Startup;
+    using OpenQA.Selenium;
+    using Selenium.Axe;
+    using Xunit;
+
+    /// <summary>
+    /// Accessibility Tests Base.
+    /// </summary>
+    [Collection("Selenium test collection")]
+    public class AccessibilityTestsBase
+    {
+        /// <summary>
+        /// Gets the base URL for the tests.
+        /// </summary>
+#pragma warning disable SA1401 // Fields should be private
```

Comment:
A project-wide StyleCop override might be appropriate rather than suppressing individual warnings?